query05.sql

```diff
@@ -3,4 +3,22 @@
 */
 
 -- Enter your SQL query here
-select ...
+CREATE TABLE durations as
```

Comment:
Creating an intermediate table is fine and makes sense (especially when doing more complicated analyses), but a [`CREATE TEMPORARY TABLE`](https://www.postgresql.org/docs/current/sql-createtable.html#SQL-CREATETABLE-TEMPORARY) might be more appropriate, as those intermediate tables wouldn't pollute your schema.